File: app/src/main/java/cn/intret/app/picgo/model/RecentRecord.java
package cn.intret.app.picgo.model;

import com.google.gson.annotations.SerializedName;

public class RecentRecord {

    @SerializedName("file_path")
    String mFilePath;

    public String getFilePath() {
        return mFilePath;
    }

    public RecentRecord setFilePath(String filePath) {
        mFilePath = filePath;
        return this;
    }

    @Override
    public String toString() {
        return "RecentRecord{" +
                "mFilePath='" + mFilePath + '\'' +
                '}';
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/user/data/RecentRecord.kt
package cn.intret.app.picgo.model.user.data

import com.google.gson.annotations.SerializedName

/**
 * Folder open history record item
 */
class RecentRecord {

    @SerializedName("file_path") internal lateinit var mFilePath: String

    var filePath: String? = null
        get() = mFilePath

    fun setFilePath(filePath: String): RecentRecord {
        mFilePath = filePath
        return this
    }

    override fun toString(): String {
        return "RecentRecord{" +
                "mFilePath='" + mFilePath + '\''.toString() +
                '}'.toString()
    }
}
